components/Report/basic_usage.rst

```diff
@@ -0,0 +1,124 @@
+Basic Usage
+===========
+
+.. note::
+    First of all you need to register your data fetcher and renderer.
+    For more detailed information go to :ref:`Registry`_.
+
+.. _Registry: http://docs.sylius.org/en/latest/components/Registry/basic_usage.html
+
+.. _component_report_data-fetcher_delegating-data-fetcher:
+
+DelegatingDataFetcher
+---------------------
+
+This service allows you to delegate fetching to your custom registered service.
+Your data fetcher class should implement `DataFetcherInterface`_.
+
+.. _DataFetcherInterface: http://api.sylius.org/Sylius/Component/Report/DataFetcher/DataFetcherInterface.html
+
+.. code:: php
```

Comment:
Are you sure? We generally use `.. code-block:: php` to have the php style, almost like in IDE, with colours :)